src/test/java/internship/framework/pages/TermsAndConditionsPage.java

```diff
@@ -0,0 +1,29 @@
+package internship.framework.pages;
+
+import internship.framework.core.PageBase;
+import internship.framework.core.utility.ProjectLogger;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static internship.framework.core.utility.ElementUtils.getElementText;
+
+public class TermsAndConditionsPage extends PageBase {
+
+    @FindBy(xpath = "//h1[@class='headline__heading']")
+    private WebElement termsAndConditions;
+
+    @FindBy(xpath = "//b[contains(text(),'Policies and Processes')]")
+    private WebElement policiesAndProcesses;
+
+    public String isTermsAndConditionsDisplayed()
```

Comment:
a better naming would be getTermsAndConditionsTitleText

Reply:
Fixed.